Silicon/TigerlakePkg/Include/TxtLib.h

```diff
@@ -0,0 +1,13 @@
+/** @file
+  TCC Config Sub Region's Struct Data
```

Comment:
Update the comments to match with TXT lib.

Reply:
done